scripts/plot_dataframe.py

```diff
@@ -142,6 +178,8 @@ def plot_analyzed_dataframe(args: Namespace) -> None:
     fig.append_trace(volume, 2, 1)
     fig.append_trace(macd, 3, 1)
     fig.append_trace(macdsignal, 3, 1)
+    fig.append_trace(trade_buys, 1, 1)
+    fig.append_trace(trade_sells, 1, 1)
```

Comment:
will this add a "enable/disable" checkbox even if it's empty? if yes, i would only add it if  db_url is set ...